Capfile

```diff
@@ -18,9 +18,13 @@ require 'capistrano/deploy'
 # require 'capistrano/rvm'
 # require 'capistrano/rbenv'
 # require 'capistrano/chruby'
-require 'capistrano/bundler'
+# require 'capistrano/bundler'
+
 require 'capistrano/rails'
 require 'dlss/capistrano'
+require 'capistrano/passenger'
+# FIXME: this isn't working for dev or stage:
+# 'SSL_connect returned=1 errno=0 state=SSLv3 read server certificate B: certificate verify failed'
```

Comment:
it has been removed.  Not sure squash is super happy, but since we're going to replace it with HoneyBadger, that's no biggie.